### ISSUE TITLE
Completely (as much as possible) replace GPG key on account

### ISSUE THREAD

**Issue by ZelnickB** [keybase/client#24594]
Hi,

I generated a key using `keybase pgp gen`, but this created a 4096-bit RSA key. I want to create an ECC key due to the security and efficiency benefits of this newer algorithm.

I am trying to completely replace my key on Keybase. I used the option in the Mac client to "drop" the key, and then I ran `keybase pgp purge` and `keybase pgp purge -p`. I now have a few concerns:

1. When I generated the key using `keybase pgp gen`, the program mentioned something about uploading the PGP key to Keybase's encrypted secret keystore, so I selected this option. How can I remove the secret key from this secret keystore?
2. How can I *revoke* my key? I know that in dropping the key, I have issued a statement in my sigchain stating that the key is not valid, but is this the same thing as a *PGP revocation* signed with the secret key? That is, if someone uploaded my key to a keyserver or downloaded my key for use outside of Keybase, can I upload a revocation certificate to the Keybase server so that they are made aware that the key is no longer used when they try to encrypt or verify something using my public key?
3. In the Keybase Mac client (under the Crypto tab), I can still sign messages, encrypt messages to myself, and decrypt them. However, on the website, when I try to encrypt a message to myself, I get an error saying that I do not have a public key on my account (I haven't yet generated my new ECC key). So, (a) how is the Keybase client signing, encrypting, and decrypting messages if I don't have a public key on my account, and (b) why is the website behaving differently? This is concerning because when I add my new key, I don't want messages encrypted with the Keybase client ciphered with the superseded key while messages encrypted on the website are ciphered with the new key.

Basically, what I am trying to ask is: how can I reset my Keybase account to the way it was before I generated the RSA key? I know that I won't be able to remove the key and its revocation from my public sigchain, but how can I indicate that this old key is never to be used again?

And, finally, how do I use `keybase pgp gen` to create a private key using a particular ECC curve? I can't find anything about `keybase pgp gen` anywhere in the documentation.

I appreciate any help that you can offer!
